views/settings/woocommerce.html

```diff
@@ -0,0 +1,34 @@
+<table class="form-table">
+  <tr>
+    <th scope="row">
+      <label for="settings[mailpoet_subscribe_old_woocommerce_customers]">
+        <%= __('Subscribe old WooCommerce customers') %>
+      </label>
+      <p class="description">
+        <%= __('Subscribe all your past customers to the “WooCommerce Customers” list because they agreed to receive marketing emails from me.') %>
```

Comment:
This sentence doesn't seem to be correct, it either should be
> Subscribe all your past customers to the “WooCommerce Customers” list because they agreed to receive marketing emails from **you**.

or

> Subscribe all **my** past customers to the “WooCommerce Customers” list because they agreed to receive marketing emails from me.